snuba/utils/streams/streaming.py

```diff
@@ -328,6 +331,8 @@ def __init__(
             ]
         ] = deque()
 
+        self.__batches_in_progress = Gauge(metrics, "batches_in_progress")
```

Comment:
Might make sense to call this `pool.tasks` or something instead? I'm not sure. I'm also open to normalizing this over the pool size so that it is a 0-1 value — `pool.utilization` or something like that.